lib/entities/content-type-fields.ts

```diff
@@ -1,19 +1,19 @@
-export interface NumRange {
+interface NumRange {
   min?: number
   max?: number
 }
 
-export interface DateRange {
+interface DateRange {
   min?: string
   max?: string
 }
 
-export interface RegExp {
+interface RegExp {
   pattern: string
   flags: string
 }
 
-export interface Validation {
+export interface ContentFieldValidation {
```

Comment:
How about `ContentTypeFieldValidation` or do we use use `ContentField` uniform to reference a content type's field?

Reply:
I'm fine with `ContentTypeFieldValidation` 👍